services/token-reward-service/.env

```
@@ -0,0 +1,16 @@
+DB_PASSWORD=password
+DB_HOST=localhost
+DB_PORT=5433
+DB_NAME=course_db
+
+WEB3_PROVIDER_URL=http://127.0.0.1:7545
+ACCOUNT_PRIVATE_KEY=0xec22a90f77ee5551ab275d723ef70c72f3adf51cba2384a55b4d6d12657e6fa6
+
+CONTRACT_ADDRESS=0x217395230cC8Fa9Cad73e1DF3732E6a32F979787
+ESCROW_CONTRACT_ADDRESS=0xa7FBAc24452dA656AEdE23279993DC90D5105239
+REWARD_DEPOSIT_ADDRESS=0xBd289d6b07Ad6A7A267F6acc9bC9f1C33E803D17
+REWARD_LISTENER_START_BLOCK=0,2
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)